scripts/enrich_rulings_ai.py

```
@@ -14,94 +15,139 @@
-    Adds tags such as "timing_window" or "cancellation_effect" if corresponding keywords are detected in the ruling text, merges them with any existing tags, and returns a sorted list.
-
-    Parameters:
-        ruling_text (str): The text of the ruling to analyze.
-        existing_tags (list[str]): A list of tags already associated with the ruling.
-
-    Returns:
-        list[str]: A sorted list of tags including both existing and newly generated tags.
+    Generates relevant tags for a ruling based on its text content using an LLM.
+    """
+    logging.info(f"AI: Generating tags for: '{ruling_text[:100]}...'")
+    prompt = f"""
```

Ruff (0.11.9)

`info()` call on root logger

(LOG015)